package.json

```diff
@@ -46,6 +47,14 @@
     "react": "^16.13.1",
     "rimraf": "^3.0.2",
     "rollup": "^2.28.2",
+    "@storybook/addon-actions": "^6.0.6",
```

Comment:
Are these storybook packages needed here?

Reply:
I dont think we're using all of those

Reply:
yep, indeed, will get them out in my next push, cheers guys 👍

Reply:
Can they go in the core package?